src/main/java/wooteco/subway/web/ExceptionController.java

```diff
@@ -0,0 +1,54 @@
+package wooteco.subway.web;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import wooteco.subway.exception.EntityNotFoundException;
+import wooteco.subway.exception.SameSourceTargetException;
+import wooteco.subway.service.member.dto.ErrorResponse;
+import wooteco.subway.web.member.InvalidAuthenticationException;
+
+/**
+ *   class description
+ *
+ *   @author ParkDooWon
+ */
+@ControllerAdvice
+public class ExceptionController {
+	@ExceptionHandler(Exception.class)
+	public ResponseEntity<ErrorResponse> handleException() {
+		return ResponseEntity
+			.status(HttpStatus.INTERNAL_SERVER_ERROR)
+			.body(new ErrorResponse("SYSTEM_ERROR"));
+	}
+
+	@ExceptionHandler(IllegalArgumentException.class)
+	public ResponseEntity<ErrorResponse> handleIllegalArgumentException() {
+		return ResponseEntity
+			.status(HttpStatus.BAD_REQUEST.value())
```

Comment:
```suggestion
			.status(HttpStatus.BAD_REQUEST)
```

따로 value를 입력하지 않아도 괜찮겠네요.

Reply:
불필요한 메서드 사용을 삭제했습니다!